lib/tasks/dev.rake

```diff
@@ -129,10 +131,25 @@ namespace :dev do
               )
               enrollment.save!
 
-              UspsInPersonProofing::EnrollmentHelper.schedule_in_person_enrollment(
-                user,
-                pii,
-              )
+              success = false
+              num_attempts = 0
+              max_attempts = 3
```

Comment:
I'm going to just go ahead and make this read from an environment variable and default to 3. We may want to tweak it as we go